blockchain/contracts/FlatNFTtheWorld.sol

```diff
@@ -1279,6 +1281,7 @@ contract NFTtheWorld {
         uint256 _dropTime,
         string[] memory _uris,
         uint256 _weiPrice,
+        uint256 _reservationTimeoutSeconds,
```

Comment:
When you change the functions, please also remember to update the documentation :)